sceptre/template.py

```diff
@@ -288,12 +290,22 @@ def _render_jinja_template(template_dir, filename, jinja_vars):
         :returns: The body of the CloudFormation template.
         :rtype: str
         """
-        logger = logging.getLogger(__name__)
-        logger.debug("%s Rendering CloudFormation template", filename)
-        env = jinja2.Environment(
-            loader=jinja2.FileSystemLoader(template_dir),
-            undefined=jinja2.StrictUndefined
-        )
-        template = env.get_template(filename)
-        body = template.render(**jinja_vars)
-        return body
+        try:
+            logger = logging.getLogger(__name__)
+            logger.debug("%s Rendering CloudFormation template", filename)
+            env = jinja2.Environment(
+                loader=jinja2.FileSystemLoader(template_dir),
+                undefined=jinja2.StrictUndefined
+            )
+            template = env.get_template(filename)
+            body = template.render(**jinja_vars)
+            return body
+
+        except Exception as e:
+            _, _, tb = sys.exc_info()
+            stack = traceback.extract_tb(tb)
```

Comment:
could we rename to `stack_trace` as `stack` is a particularly overloaded word in this project.

Reply:
Good point, I'll do that